examples/image_series_annotator_app.py

```diff
@@ -23,25 +27,53 @@ def segment_wigdet(v: Viewer):
     v.layers["segmented_object"].refresh()
 
 
-def image_series_annotator(image_paths, embedding_save_path, output_folder):
-    global PREDICTOR
+def increase_brush_size(v):
+    """
+    Change brush size in layer segmented object to correct the segmentation result manually
+    """
+    if v.layers.selection.active == v.layers["segmented_object"]:
+        v.layers["segmented_object"].brush_size += 1
+
+
+def decrease_brush_size(v):
+    """
+    Change brush size in layer segmented object to correct the segmentation result manually
+    """
+    if v.layers.selection.active == v.layers["segmented_object"]:
+        current_size = v.layers["segmented_object"].brush_size
+        if current_size > 1:
+            v.layers["segmented_object"].brush_size -= 1
+
+
+def activate_segmented_object():
+    v.layers.selection.active = v.layers["segmented_object"]
+
+
+def image_series_annotator(embedding_save_path, output_folder):
+    global PREDICTOR, v, image_paths
 
     os.makedirs(output_folder, exist_ok=True)
 
     # get the sam predictor and precompute the image embeddings
     PREDICTOR = util.get_sam_model()
-    images = np.stack([imageio.imread(p) for p in image_paths])
-    image_embeddings = util.precompute_image_embeddings(PREDICTOR, images, save_path=embedding_save_path)
-    util.set_precomputed(PREDICTOR, image_embeddings, i=0)
+    image = imread(image_paths[0])
+
+    if len(image.shape) == 3:
+        image_embeddings = util.precompute_image_embeddings(PREDICTOR, rgb2gray(image))
```

Comment:
You need to pass the argument `ndim=2`  to the precompute function, then it also works with rgb data.
This tells `precompute_image_embeddings` that the input has 2 spatial dimension and it will interpret the last axis as RGB then.